avahi-daemon/main.c

```diff
@@ -705,6 +705,40 @@ static int load_config_file(DaemonConfig *c) {
                     for (t = e; *t; t++)
                         c->server_config.deny_interfaces = avahi_string_list_add(c->server_config.deny_interfaces, *t);
 
+                    avahi_strfreev(e);
+                } else if (strcasecmp(p->key, "allow-addresses") == 0) {
+                    char **e, **t;
```

Comment:
and again, this seems like hot candidate for ``avahi_address_list_add`` static function called for two different pointers.